### PR TITLE
portion of UCP related fixes for PGI

### PR DESCRIPTION
**Pull request by hoopoepg**
- first portion of UCP related fixes for PGI compiler support

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12537/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 15-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/74d471c91b214b3873d1be4bcff0d4dc50b411fb/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/046ac306eb432dbc4dc95294d045a785880f0152/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/f0a699aad93c3a3b441795c7a7cda2636f18ed7e/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/7c0d551a90d3f3c488b35d57922be2d7f959872c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/fc5c1397a3a91f1344dff554b5dd045e2db484d8/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/51f3d1ac6f69ed98dba47b00e8feae4137e40638/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/9c5a79c450653b147f295b44d7138acffed3b9cb/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/17041b1f8ff6488b254c84f91ce583ddba2a58e0/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/aa25f713e8e731924c2dd115a61fc3dac7c19a52/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/b1c541ac4d1d54ed0128e1ceda9a2d123f2ef723/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/a4045581ec505b5eee38ce55b6f1f9c104b6e7fb/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/f682125fab440a8b223d0cc4b658282879f4bf23/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/a02a06718c89784a386ff6fba266d22add2ac1b1/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/346657e1ac638debae6ff51f854a3d9fe883a842/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/386bacd0daef7db9e8622afd82e047f905267f21/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/7b8c4022c313d27c3e6237c30594d5f615a50831/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/c4233f577d6968e49ecf54bb8bcffdafcc80c87e/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/4b36c9158e00e12b816876259062c1b03a77cf1b/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/5f7a64be84330ff7b82e632d33e4f0c60feb62f5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/407b0162b0764cd4dc81bd39b20480d0b113458f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/95e99b7d363e4d012a0a6455daf132fb221965f8/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/649937c1cf0947205cccffd6cb4b768dbe1abd15/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/e3d823fd4a98364a8f97e438877491b3e8d356ce/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/58bd95234d19a754e12304fa72fb6141d40f0571/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/799bf2c7324322b8d983042d8b47218584de71a1/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/a31081115dd87cd350787c1b918c3111bafdcd09/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/495a2304f721be16429a8376dcbe3aaa39c45c71/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/6775f729a90ec359468a852a2bac3991b15fc1d7/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/62b9d420cc69665c010c9561f6122321/raw/80a1a73f1687bf685d015c40b74577ba469512b7/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/ucp/core/ucp_rkey.c [hunk 1]
```diff
@@ -296,15 +297,13 @@ void ucp_rkey_dump_packed(const void *rkey_buffer, char *buffer, size_t max)
...
-
-    rkey_buffer += sizeof(uint8_t);
+    md_map    = *(ucp_md_map_t*)(rkey_buf);
+    rkey_buf += sizeof(ucp_md_map_t) + sizeof(uint8_t);
 
     first = 1;
     ucs_for_each_bit(md_index, md_map) {
-         md_size      = *((uint8_t*)rkey_buffer);
-         rkey_buffer += sizeof(uint8_t);
+         md_size   = *((uint8_t*)rkey_buf);
```
Comment (brminich) [inline src/ucp/core/ucp_rkey.c]: no casting needed?

File: src/ucp/core/ucp_worker.c [hunk 2]
```diff
@@ -67,7 +54,20 @@ static ucs_stats_class_t ucp_worker_tm_offload_stats_class = {
         [UCP_WORKER_STAT_TAG_OFFLOAD_BLOCK_NO_IFACE]   = "block_no_iface",
         [UCP_WORKER_STAT_TAG_OFFLOAD_RX_UNEXP_EGR]     = "rx_unexp_egr",
         [UCP_WORKER_STAT_TAG_OFFLOAD_RX_UNEXP_RNDV]    = "rx_unexp_rndv",
-        [UCP_WORKER_STAT_TAG_OFFLOAD_RX_UNEXP_SW_RNDV] = "rx_unexp_sw_rndv",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_RX_UNEXP_SW_RNDV] = "rx_unexp_sw_rndv"
+    }
+};
+
+static ucs_stats_class_t ucp_worker_stats_class = {
```
Comment (brminich) [inline src/ucp/core/ucp_worker.c]: why need to move?

File: src/ucp/core/ucp_rkey.c [hunk 3]
```diff
@@ -44,14 +44,14 @@ void ucp_rkey_packed_copy(ucp_context_h context, ucp_md_map_t md_map,
...
-    void *p = rkey_buffer;
+    uint8_t *p = rkey_buffer;
     unsigned md_index;
     size_t md_size;
 
     *(ucp_md_map_t*)p = md_map;
     p += sizeof(ucp_md_map_t);
 
-    *((uint8_t *)p++) = mem_type;
+    *(p++) = mem_type;
```
[the remark this answers is not in the document]
Comment (hoopoepg) [reply src/ucp/core/ucp_rkey.c]: yep, removed

File: src/ucp/core/ucp_rkey.c (hunk 1 above)
Comment (hoopoepg) [reply src/ucp/core/ucp_rkey.c]: right, removed too

File: src/ucp/core/ucp_worker.c (hunk 2 above)
Comment (hoopoepg) [reply src/ucp/core/ucp_worker.c]: PGI compiler bug, reported to PGI compiler team

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12547/ for details (Mellanox internal link).

**Comment by brminich** [comment]
plz, resolve conflict

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 17-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/02909a5142af0e1fc60761e17db72380f53571e2/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/444e797d109da05ef2fbbbf7325815ab2908a6ff/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/42a9cb44f1c97f41cd0d78ffc5cc1100dfdd7324/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/b8a243d2b8ac5c105a7e06ac873595100874a910/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/3bb44859d9d9ab9a7e28e6b96dc18612ed6a5cd0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/31fe4fba07977f0cd1b5749f2cd3b1ba60bfdd0b/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/57ef001d2ddc75918df184ca6ad5527e3181b50a/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/8adcffef2d9bb257423e8a7a9a2778dac7ef8a28/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/c93ff2cfdd398f2857fbb1abb316f0fec0fcb2bb/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/a06a00463a75dba3f3f7838bd67ff3150d6509d0/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/612f1e0ea109219697b728a54bc41710d177676a/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/dafac8f208d07d4391316e174fbeb32aa84ccafc/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/61dc8e702ff8720695729b4b5a26e08e061e4fa5/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/a235ed1b8905b7c5676b4f1f41beb9c255ddbf8a/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/710d975e659dafe0fe5f4c1a1de94fe263d130fe/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/7125ef761cf4649e58a1545cd2bd121e8eb44cba/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/a546427451ab4a77f2d68d185e39f5e0d9806b10/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/cb48f9b593be7c95ef047cc1e22d6ecdfc4e8c28/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/b2a6038ad4bb72cf44c4804f65e509ed2cbcb422/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/5d177c4dd44a391b9d6d4f6ea109a922e52f4537/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/afc67645e415331745f669dd90bc7eb7a408b586/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/d985d93654bfa5ac904183597c1f8b0cfba0288e/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/49d180f6509f64eb36ac57f0e362b2c42b18b5da/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/5e2d3ec09a5299fbb0e4dcec787b98ee19d45d40/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/82ef3cd7d8b400d2859f240cf8a2acedee0e78f0/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/1621e672be47ec853c7b1d0948be3777f7ecc362/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/d98c34bf1d65fc57b12695a6c2c62cb6cc718deb/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/92f7b32bf4eac8861078c16fb9c26b1e59f68fba/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/38f5b9fb95ad923d49fd0a38863c5075/raw/584be1fa9aab14484001296e269d590a4f125851/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12564/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12580/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 18-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/6585e424221908e8a1d925f8cb90d89dd85496fb/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/b1a0f90502a33344b93542f8202d30c4a14c1e35/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/e42e9f9cd019e99e9012e0e6bab1cf1a9064a740/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/8e45db5b0bcd952411a25a1da80dff01d1fe79e9/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/a218b9da93353799006e216a6c49b09ff249b578/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/510ddd51b008a50709452ec835d07a3df4880507/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/6cbd688d3fdb004116ca768f5a170c8b9123d6a6/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/325ac0fe59185c1ea6b0775569d8119e8cc176b6/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/e1a8d910c36d89cb61be5a5a38f56c1d1067fc7d/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/18e76426572c0074e65053fa5e967c57cfcacec1/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/f5fa5735042c5cc70abf7c3a42a1fb2cff3c31a9/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/857e87a2cc7c378f6fe520865cddb1d37e831b55/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/e6eb30843e0046a79de454468a9301f5f89b0a6e/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/c4350138503b763d119431327e0f6ee6b95c5ca6/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/8637b176a2123a1abe47c9130d79ca7d309b4be3/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/f99626b20a77a4b4d8405dbf1ab7410441ec2e22/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/60a6cd0d755f670c4e9f1fc9dd13291951b06188/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/6dbe798cc596074b1ca3e6d8564393dbc4d42b86/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/68383ffc582c75a7fe52f8e1b45c8880dac02c78/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/9de23dfe0206ae220121dcaaf8bd9cc8f84792d6/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/430d10b34297332ea79263ff1dd9511622269adc/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/fce5fa817e574816ca2539970c1d26a1aac40149/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/a35e10f47e34875ae1c624d896cdc2ada2d7cc1d/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/fef211d8bd0747e695543b4ff77719aebd81c819/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/c5bc48e2ca099fef9c82ce25f4d1a97caa088c58/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/c0691b056fb93a8e880fe66d4fb0720b0e522807/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/053e3045fb8fda98ff60c9846d162a5f32ea05c0/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/e8c163acc51fcdb996668dd69960ee50925ad6dc/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7e6bc553a6bac219fd8de058dfbaa9ae/raw/e4d1a205442dad505f02795f462309f567fb0b8a/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by hoopoepg** [comment]
bot:bgate:retest

**Comment by brminich** [comment]
we need to add PGI checks to our CI asap. Looks like it is very easy to break PGI compilation

**Pull request merged**